app/helpers/record_counts_helper.rb

```diff
@@ -0,0 +1,25 @@
+module RecordCountsHelper
+  def stylised_recent_additions(additions, options = {})
```

Comment:
Assignment Branch Condition size for stylised_recent_additions is too high. [31.46/15]<br>Method has too many lines. [19/10]